about.md

```diff
@@ -3,6 +3,95 @@ layout: page_md
 title: "About Us"
 tagline: Some information about the Jupyter Project and Community
 permalink: /about
+distinguished_special_election:
+  - name: Cameron Oelsen
+    photo: Cameron_Oelsen.png
+    personal: https://www.cameronoelsen.com/
+    linkedin: https://linkedin.com/in/cameronoelsen
+    gh_handle: CameronOelsen
+  - name: Matthew Seal
+    photo: Matthew_Seal.jpg
+    gh_handle: MSeal
+    linkedin: https://linkedin.com/in/matthew-seal-70a07932/
+    twitter_handle: codeseal
+  - name: Grant Nestor
+    photo: Grant_Nestor.jpg
+    gh_handle: gnestor
+    linkedin: https://linkedin.com/in/grantnestor/
+    twitter_handle: senornestor
+  - name: Ian Rose
+    photo: Ian_Rose.jpg
+    gh_handle: ian-r-rose
+    twitter_handle: IanRRose
+    personal: https://ian-r-rose.github.io/
+  - name: Douglas Blank
+    photo: placeholder.png
```

Comment:
```suggestion
    photo: https://avatars.githubusercontent.com/u/168568
```

Reply:
nevermind, you use relative URL so will need to copy it.

Reply:
Ok, so you're set? Thanks!

Reply:
We didn't want to use an avatar without permission @dsblank! I'm happy to update it now with your GH avatar!

Reply:
Perfect! Sorry if I missed a request!